geomesa-core/src/main/scala/org/locationtech/geomesa/core/util/SftBuilder.scala

```diff
@@ -149,7 +148,7 @@ object SftBuilder {
 
   // Note: not for general use - only for use with SimpleFeatureTypes parsing (doesn't escape separator characters)
   def encodeMap(opts: Map[String,String], kvSep: String, entrySep: String) =
-    opts.map { case (k, v) => (k + kvSep + v) }.mkString(entrySep)
+    opts.map { case (k, v) => k + kvSep + v }.mkString(entrySep)
```

Comment:
Yes, I have been making changes to our SimpleFeatureType factories and builder patterns - this one snuck in because intellij auto-corrected the code.